Doc/library/tarfile.rst

```diff
@@ -1346,6 +1346,16 @@ parameter in :meth:`TarFile.add`::
     tar.add("foo", filter=reset)
     tar.close()
 
+How to create and write the archive to stdout using
+:data:`sys.stdout.buffer <sys.stdout>` in the *fileobj* parameter
+in :meth:`TarFile.add`::
+
+    import tarfile
+    import sys
+    files = ["foo.txt", "bar.txt", "quux.txt"]
+    with tarfile.open("sample.tar.gz", "w|gz", fileobj=sys.stdout.buffer) as tar:
+        for file in files:
+            tar.add(file)
```

Comment:
For consistency with the earlier examples.
```suggestion
    import sys
    import tarfile
    with tarfile.open("sample.tar", "w", fileobj=sys.stdout.buffer) as tar:
        for name in ["foo", "bar", "quux"]:
            tar.add(file)
```

Reply:
Updated